Azure/azure-cli
# [ams] update python to track 2

**Description**
Updating the ams python cli to track 2. 

**Testing Guide**
No changes to testing

---

This checklist is used to make sure that common guidelines for a pull request are followed.
